contracts/strategies/stkCvxFXS/Strategy.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../../../interfaces/IBooster.sol";
+import "../../../interfaces/IStrategyOracle.sol";
+import "../../../interfaces/IGenericVault.sol";
+import "../../../interfaces/ICvxFxsStaking.sol";
+import "../../../interfaces/IHarvester.sol";
+
+contract stkCvxFxsStrategy is Ownable {
+    using SafeERC20 for IERC20;
+    address public constant CVXFXS_TOKEN =
```

Comment:
Consider changing the type to `IERC20` since that's what it's always casted to and its original `address` type is never utilized in this contract.